static/docs/commands-reference/status.md

```diff
@@ -53,19 +53,40 @@ This says that no differences were detected, and therefore that no stages would
 be run again if `dvc repro` were executed.
 
 If instead, differences are detected, `dvc status` lists those changes. For each
-DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
-differ are listed. For each item listed, either the file name or the checksum is
-shown, and additionally a status word is shown describing the change:
+DVC-file (stage) with differences, the changes in both _dependencies_ and/or
+_outputs_ that differ are listed. For each item listed, either the file name or
+the checksum is shown, and additionally, a status word is shown describing the
+changes in checksum or status for both local and cloud workspace. Given list
+provides a reference to both, the status of the DVC-file and differences in the
+DVC-files:
 
 - For the local workspace:
-  - _changed_ means the file has changed
+
+  - _changed checksum_ means actual checksum of the DVC-file doesn't match the
+    one specified in it in DVC-files
+  - _always changed_ means that this is a special DVC-file with no dependencies,
```

Comment:
for all the items, there should be some rule about punctuation at the end. Most likely ; will do. @jorgeorpinel do you know about this?

Reply:
I haven't standardized this yet but actually I think we mostly use `.` throughout the docs since our bullet lists tend to contain long paragraphs.

Reply:
Also on this line, note that "special DVC-file with no dependencies" is also known as an "orphan" stage file, as mentioned in a few docs:

![image](https://user-images.githubusercontent.com/1477535/60850320-09a89200-a1ac-11e9-837f-4d9ceb17136d.png)

Reply:
good points. I think the rule should be - if items are short one sentence long, and start from lowercase then we should be using `;` (or nothing?), otherwise `.` make sense.

Reply:
OK, I just standardized it in 3134296. Long list items (paragraphs) and all cmd ref/ config option list items end in period `.` and the list items are separated by a new line. Shorter lists (a sentence or less) end in semicolon `;` and don't have extra new lines in between. I also fixed the indentation of a bunch of lists.

The part about starting in lower case varies though. All the cmd ref option list items start in lower case even when they tend to be full paragraphs for ex., since they are preceded by a `-` dash. E.g.

```
- `-q`, `--quiet` - do not write anything to standard output. Exit with 0 if no
  problems arise, otherwise 1.
```